backend/app/auth/service.py

```
@@ -289,38 +289,46 @@ async def refresh_access_token(self, refresh_token: str) -> str:
-        """
-        from app.auth.security import verify_token
+    Returns:
+        The user document corresponding to the token's subject.
 
-        payload = verify_token(token)
-        user_id = payload.get("sub")
+    Raises:
+        HTTPException: If the token is invalid or the user does not exist.
+    """
+    from app.auth.security import verify_token
+    from bson import ObjectId
```

Ruff (0.12.2)

Redefinition of unused `ObjectId` from line 16

Remove definition: `ObjectId`

(F811)